src/types.js

```diff
@@ -153,6 +153,8 @@ export type Dependency = {
   latest: string,
   url: string,
   hint: ?string,
+  range: string,
+  latestPattern: string,
```

Comment:
(+ in the rest of the code)

I feel like the name isn't very clear - the difference between `latest` and `latestPattern` should be explained somewhere